Description: Remove dep on zstd feature thin, unnecessary using Debian packaged zstd
Last-Update: 2023-03-02
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -152,7 +152,6 @@
 default = [
     "flate2/zlib",
     "zip/deflate-zlib",
-    "zstd/thin",
 ]
 
 [target."cfg(not(unix))".dependencies.is_executable]
